extras/printschema.py

```diff
@@ -0,0 +1,45 @@
+import copy
```

Comment:
Hmm, I'm a git newbie.  I could just delete the file from HEAD, but that wouldn't make commit 62d2b30 disappear.  Is that OK or did you want something fancier?

I'm using genson to analyze large (tens of KBs, some up to 1MB) data objects and using the generated schemas as documentation.  With the python module I can put all schemas in a consistent format -- type first, then required, then properties,  required and basic types on one line, nested types indented on multiple lines, etc).  The schema-aware pprinter will handle any dict, but it does a better job than generic pprinters on dicts that are schemas.  You don't need to include it with genson; I just thought someone might be interested.